cinemy/login/models.py

```diff
@@ -0,0 +1,5 @@
+from rest_framework import authentication
+
+
+class BearerAuthentication(authentication.TokenAuthentication):
```

Comment:
usually, authentication classes are placed in new files named `authentication`. In models you only place entities that are translatable to a db table (that inherit `models.Model`)